actions/mouths-of-watersheds/mouths-of-watersheds.py

```diff
@@ -0,0 +1,52 @@
+import argparse
+import csv
+from netCDF4 import Dataset
+from shapely.geometry import shape
+import geojson
+from mow.helpers import *
+from mow.geo_data_grid_2d.vic import VicDataGrid
+
+parser = argparse.ArgumentParser('Output a CSV file with the lon lat of the mouth of each watershed')
+parser.add_argument('nc', help='an RVIC-netCDF file')
+parser.add_argument('geojson', help='a geoserver output file describing the watersheds')
+parser.add_argument('csv', help='the csv file to output the results to')
+args = parser.parse_args()
+nc = Dataset(args.nc, "r")
+flow_direction = VicDataGrid.from_nc_dataset(nc, "flow_direction")
```

Comment:
Use a context manager (with statement) here. Then the file is always guaranteed to be closed after use.

```
with Dataset(...) as nc:
    flow_direction = ...
```